samples/tacos/src/main/kotlin/com/slack/circuit/tacos/repository/IngredientsRepository.kt

```diff
@@ -0,0 +1,117 @@
+package com.slack.circuit.tacos.repository
+
+import com.slack.circuit.tacos.model.Diet
+import com.slack.circuit.tacos.model.Ingredient
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.coroutines.delay
+import kotlin.random.Random
+
+interface IngredientsRepository {
+  suspend fun getFillings(): ImmutableList<Ingredient>
+  suspend fun getToppings(): ImmutableList<Ingredient>
+}
+
+object IngredientsRepositoryImpl : IngredientsRepository {
+  override suspend fun getFillings(): ImmutableList<Ingredient> {
+    delay(Random.nextLong(0, 5000))
```

Comment:
Why the random delay?

Reply:
There are loading states for most steps. I think I initially added this so they were visible